third_party/tsl/third_party/gpus/cuda/build_defs.bzl.tpl

```diff
@@ -104,9 +104,16 @@ def if_cuda_newer_than(wanted_ver, if_true, if_false = []):
     wanted_major = int(wanted_ver.split('_')[0])
     wanted_minor = int(wanted_ver.split('_')[1])
 
-    configured_version = "%{cuda_version}"
-    configured_major = int(configured_version.split('.')[0])
-    configured_minor = int(configured_version.split('.')[1])
+    # Strip "64_" which appears in the CUDA version on Windows.
```

Comment:
This is mostly self-explanatory. In Windows, the CUDA version looks something like `64_121`. There are other parts of the build that already handle such version numbers but it was not being handled properly here.